docs/gmrq.rst

```diff
@@ -1,8 +1,33 @@
 .. _gmrq:
 
-Model Selection (GMRQ)
+Model Selection using GMRQ
```

Comment:
Check out the sidebar on msmbuilder.org. I like the idea of each page title being of the form
> High level description (technical acronym)

We tell people a lot about "GMRQ"; It should be immediately apparent where they can get a description of what this mysterious 4LA is

Also, if you want to change the title, adjust the underbar to be the same width